examples/benchmarking.ipynb

```diff
@@ -0,0 +1,180 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline"
```

Comment:
Is this supposed to show up at the top? (the answer might be yes, I'm not very familiar enough with jupyter notebooks)

Reply:
I remember us talking about it at one point in the past, but I can't remember why it's here. Can you explain it for me?

Reply:
We mentioned how the automated notebooks had this at the top. But from what I can tell, this does not serve a purpose for us, so I'll remove this cell.